root/usr/share/nethesis/NethServer/Module/Mail/Domain/Modify.php

```diff
@@ -43,15 +45,20 @@ public function initialize()
             array('Description', Validate::ANYTHING, Table::FIELD),
             array('TransportType', Validate::ANYTHING, Table::FIELD),
             array('DisclaimerStatus', Validate::SERVICESTATUS, Table::FIELD),
+            array('OpenDkim', Validate::SERVICESTATUS, Table::FIELD),
         );
 
         $this->declareParameter('DisclaimerText', $this->createValidator()->maxLength(self::DISCLAIMER_MAX_LENGTH), $this->getPlatform()->getMapAdapter(
                 array($this, 'readDisclaimerFile'), array($this, 'writeDisclaimerFile'), array()
             ));
 
+        $this->declareParameter('DkimKey', $this->createValidator()->maxLength(self::DKIM_MAX_LENGTH),$this->getPlatform()->getMapAdapter(
```

Comment:
Let's set it to a large enough value, say 10000.

Reply:
done